adding section for service map availability info (#17977)
DataDog/documentation: content/en/tracing/services/services_map.md

Monitors are not constrained to APM monitors. The service tag, described above, can be used to associate any monitor type with a service.

### Availability

> Service Map is rendered based on complete traces that include the root spans. When some spans are missing during the query window you specify, the map view may be unavailable for that time period. A common cause is when spans are dropped due to [APM connection errors][6].

Reviewer: ```suggestion
The Service Map is rendered based on complete traces that include the root spans. When some spans are missing during the query window you specify, the map view may be unavailable for that time period. This may happen when [APM connection errors][6] occur and spans get dropped.
```
